client/src/components/ProjectWizard/RuleCalculation/RuleCalculation.js

```diff
@@ -325,7 +345,13 @@ const RuleCalculation = ({
             data-class={classes.tooltip}
           >
             {name}
-            <ToolTip tipText={description} />
+            <ToolTipIcon
```

Comment:
![image](https://user-images.githubusercontent.com/45305029/105139126-b99dd180-5aaa-11eb-8d1d-ceb326d40ffb.png)
There shouldn't be a tooltip for the parking provided/baseline.  It should only show a tooltip IF there's a description.  So since the bottom doesn't have a description, the icon should not be showing.